Python/YouTube_Trending_Analysis/YT_Trending_Analysis.py

```diff
@@ -0,0 +1,45 @@
+import json
+import time
+import pandas as pd
+from selenium import webdriver
+# driver is our way to access the webpages
+# This code is meant to be run using the Google Chrome browser. Any other browser would require you to change the following function. 
+# eg. webdriver.Firefox for the Firefox browser.
+driver = webdriver.Chrome()
+driver.implicitly_wait(5)
+baseUrl = "https://www.youtube.com/feed/trending"
+all_details = []
+
+# This returns a list of links to all the trending videos
+def getVideoURLs():
+    driver.get(baseUrl)
+    time.sleep(3)
+    allVideoList= driver.find_elements_by_css_selector("#video-title")
+    links = list(dict.fromkeys(map(lambda a: a.get_attribute("href"),allVideoList)))
+    return links
+
+# This returns the specified details about the input video
+def getVideoDetails(video):
+	driver.get(video)
+	time.sleep(1.5)
+	vname = driver.find_element_by_xpath('//*[@id="container"]/h1/yt-formatted-string').text
+	vviews = driver.find_element_by_xpath('//*[@id="count"]/yt-view-count-renderer/span[1]').text.split()[0]
```

Comment:
Remove commas from the number of views and store them as int

Reply:
Done